data_visualization_in_seaborn/data_visualization_seaborn.md

```diff
@@ -0,0 +1,684 @@
+```
+
+![Line plot with timepoint on the x-axis and signal on the y-axis, and two lines: a solid blue line for event = stim and a dashed yellow line for event = cue.](media/seaborn_line_2.png)
+
+### Using facets to compare plots
+
+If you want to make more than one version of a similar plot, consider using facets.
+
+In the current example, let's say we wanted to create two versions of the signal line plots for the two types of events: one showing the response in the parietal region, and one showing the response in the frontal region.
+
+You can add facets to a `seaborn` plot by adding an argument for either `col` (for columns) or `row` (for rows) --- or both, if you want to show several different facets.
+
+```python
+sns.relplot(x="timepoint", y="signal", hue="event", style="event", col = "region", kind="line", data=fmri)
+```
+
+![Two line plots, side by side, with timepoint on the x-axis and signal on the y-axis, and each showing two lines: a solid blue line for event = stim and a dashed yellow line for event = cue. The plot on the left says "region = parietal" at the top, and the plot on the right says "region = frontal" at the top.](media/seaborn_line_3.png)
+
+<div class = "learnmore">
+For many more examples of line plots, see the [seaborn relplot tutorial section on line plots](https://seaborn.pydata.org/tutorial/relational.html#emphasizing-continuity-with-line-plots).
+</div>
+
+### Quiz: Line Plots
+
+True or False: You can create both scatter plots and line plots with the same function in `seaborn`.
+
+[(X)] TRUE
+[( )] FALSE
+****
+We used just one function, `relplot`, to make both scatter plots and line plots. Under the hood, there are actually two separate functions being used --- `scatterplot()` and `lineplot()`, which are equivalent to `relplot()` with `kind = "scatter"` (the default) and `kind = "line"`, respectively --- but it's nice to be able to stick to the higher-level `relplot` function for simplicity.
+
+Because they're both called by the `relplot` function, you can use the same additional variables (e.g. `hue`, `style`, `col`) applied to either scatter or line plots, and they work in much the same way.
+****
+
+True or False: Line plots are appropriate in most situations in which a scatter plot would work, so you can general pick whichever one you prefer.
+
+[( )] TRUE
+[(X)] FALSE
+****
+There are very few situations in which a scatter plot and line plot are equally appropriate.
+
+Line plots emphasize continuity and because of that are rarely used except for with data that have some kind of time component. Scatter plots are much more flexible and can work to show the relationship between almost any two continuous variables.  
+****
+
+What argument do you add to include facets in a `seaborn` plot?
+
+[[col, row]]
+<script>
+  let input = "@input".trim();
+  /col|row/i.test(input);
+</script>
+****
+You can add facets to a plot by including the arguments for `col`, `row`, or both.
+****
+
+## Trend Lines
```

Comment:
Yeah, I see what you mean, thanks! For now, I'm going to keep the ordering as-is (I prefer having trend lines at the end for the reasons you state, and I don't think it will be too taxing for them to switch between datasets).